src/modules/sign/components/gitSignIn.jsx

```diff
@@ -7,21 +7,21 @@ import activateGitlab from "../../../../src/assets/svg/gitlabActive.svg";
 function GitSignIn() {
   const [githubActive, githubSetActive] = useState(false);
   const [gitlabActive, gitlabSetActive] = useState(false);
-  const githubActiveStyles = githubActive ? {border: "1px solid black"} : {};
-  const gitlabActiveStyles = gitlabActive ? {border: "1px solid black"} : {};
+  const githubActiveStyles = githubActive ? {border: "1px solid black", cursor: "pointer"} : {};
```

Comment:
These styles can be added to the buttons using the `:hover` pseudo-class. It could be done this way:

```css
.button:hover {
    border: 1px solid black;
    curosr: pointer;
}
```
Then pass thee class name to the login buttons. With this, you won't need to create `githubActiveStyles` and `gitlabActiveStyles` objects anymore.